cmake/build_mingw32.sh

```diff
@@ -8,7 +8,7 @@ else
 	CMAKE_OPTS="$CMAKE_OPTS -DLMMS_BUILD_MSYS=1"
 fi
 
-export PATH=$PATH:$MINGW/bin
+export PATH=$MINGW/bin:$PATH
```

Comment:
Actually might fix it.  I made the change because the right pkg-config wasn't being found so it could find libsndfile.  I got a successful build after that,  and I wasn't aware of #2951.